uroot/root.go

```diff
@@ -38,6 +38,12 @@ type file struct {
 }
 
 // TODO: make this a map so it's easier to find dups.
+type dev struct {
+	name    string
+	mode    os.FileMode
+	magic   int
```

Comment:
done. I used magic as it's a bit nasty how it works, over the years the docs and actual value change.